media-libs/liblastfm/liblastfm-1.0.9.ebuild

```diff
@@ -57,14 +57,14 @@ src_configure() {
 		# demos not working
 		local mycmakeargs=(
 			-DBUILD_DEMOS=OFF
-			$(cmake-utils_use_build fingerprint)
-			$(cmake-utils_use_build test TESTS)
+			-DBUILD_FINGERPRINT=$(usex fingerprint)
+			-DBUILD_TESTS=$(usex test)
 		)
 		if [[ ${MULTIBUILD_VARIANT} = qt4 ]]; then
 			mycmakeargs+=(-DBUILD_WITH_QT4=ON)
 		fi
 		if [[ ${MULTIBUILD_VARIANT} = qt5 ]]; then
-			mycmakeargs+=(-DBUILD_WITH_OT4=OFF)
+			mycmakeargs+=(-DBUILD_WITH_QT4=OFF)
```

Comment:
Please only this change. The other changes are only cosmetic which should be done in a revision bump or version bump.